### ISSUE TITLE
Chyba v názvu 

### ISSUE THREAD

**Issue by sebruant** [Sumbera/iKatastr.cz#37]
Maksičky mezi Sadkem a Korouhví nejsou Maxičky 

https://www.ikatastr.cz/#kde=49.67918,16.21689,16&info=49.2073,16.48602&mapa=zakladni&vrstvy=parcelybudovy

**Comment by Sumbera** [comment]
ano, z toho duvodu je tam geograficka mapa: https://www.ikatastr.cz/#kde=49.677966,16.225379,17&info=49.2073,16.48602&mapa=geograficka&vrstvy=parcelybudovy
